src/app/camera.cpp

```diff
@@ -21,19 +22,61 @@ void Camera::setDirection(const glm::vec3 &new_direction) {
     direction_m = glm::normalize(new_direction);
 }
 
+void Camera::moveUp(const float speed) {
+    position_m += speed * screen_up_m;
+}
+
+void Camera::moveDown(const float speed) {
+    position_m -= speed * screen_up_m;
+}
+
+void Camera::moveLeft(const float speed) {
+    position_m += speed * glm::normalize(glm::cross(screen_up_m, direction_m));
+}
+
+void Camera::moveRight(const float speed) {
+    position_m += speed * glm::normalize(glm::cross(direction_m, screen_up_m));
```

Comment:
Consider removing glm::normalize, as we know that screen_up_m and direction_m always normalized, therefore, their cross-product will be normalized as well
```suggestion
    position_m += speed * glm::cross(direction_m, screen_up_m);
```

Reply:
Fixed in 4163279